base/fhs-validator.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOTDIR="${1:-/rootfs}"
+
+# Make sure symlinks from FHS did not get overwritten
+for EXPECTED_SYMLINK in bin lib lib64 usr/lib64 sbin usr/sbin; do
+    if [[ -e "${ROOTDIR}/${EXPECTED_SYMLINK}" && ! -L "${ROOTDIR}/${EXPECTED_SYMLINK}" ]]; then
```

Comment:
I guess this might falsely report okay if package creates its own symlink for example. Perhaps if this script is used without FHS in the first layer it should check that package does not have any of the paths which are symlinks under its root, something like extensions-validator does (but let's make it a denylist here)

Reply:
good catch